span.go

```diff
@@ -90,39 +107,20 @@ type Span interface {
 
 	// Provides access to the Tracer that created this Span.
 	Tracer() Tracer
+
+	// DEPRECATED
```

Comment:
godoc actually has an idiomatic way of declaring an identifier as deprecated:

> To signal that an identifier should not be used, add a paragraph to its doc comment that begins with "Deprecated:" followed by some information about the deprecation.

The note being in a separate paragraph also implies that the deprecated identifiers should still carry meaningful (most likely their old, pre-deprecation) documentation.